contributors.html

```diff
@@ -96,21 +95,15 @@
 	<div class="navbar-fixed">
 		<nav>
 			<div class="nav-wrapper">
-				<ul class="languagepicker roundborders large">
-					<li>
-						<a class="lang" key="language">Select your language</a>
-					</li>
-					<a class="translate" id="en">
-						<li>
-							<img src="icons/UnitedKingdom.png" width=7% height=7%/> English
-						</li>
-					</a>
-					<a class="translate" id="si">
-						<li>
-							<img src="icons/SriLanka.png" width=7% height=7%/> Sinhala
-						</li>
-					</a>
-				</ul>
+        <div class="languagepicker roundborders large">
+				  <a class="lang" key="language">Select your language</a>
+					  <a class="translate translate-language-choice" id="en">
+						  <img src="icons/UnitedKingdom.png" alt="" class="translation-flag" /> English
```

Comment:
why `alt=""` when it's blank ?

Reply:
because otherwise we get an error: https://travis-ci.org/fossasia/labyrinth/builds/328813163#L638